src/packagedcode/npm.py

```diff
@@ -500,6 +500,48 @@ def update_workspace_members(cls, workspace_members, codebase):
         for member in workspace_members:
             member.save(codebase)
 
+class NpmrcHandler(BaseNpmHandler):
+    datasource_id = 'npmrc'
+    path_patterns = ('*/.npmrc',)
+    default_package_type = 'npm'
+    default_primary_language = None
+    description = 'npm .npmrc configuration file'
+    documentation_url = 'https://docs.npmjs.com/cli/v11/configuring-npm/npmrc'
```

Comment:
```suggestion
    documentation_url = 'https://docs.npmjs.com/cli/configuring-npm/npmrc'
```
this would pin the link to this version and will soon be outdated. I see some other npm doc URLs also have versions, can you also remove those in the PR?